.github/workflows/ci.yaml

```diff
@@ -5,9 +5,54 @@ on:
   pull_request:
     branches: [ "main" ]
 
+env:
+  DOCKER_BUILDX_PLATFORMS: linux/amd64,linux/arm64
+
 jobs:
+  # Make sure that Kubebuilder autogenerated files are up to date. 
+  kubebuilder-diff:
+    name: Kubebuilder Diff
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.19 # Use the latest
+    - name: Generate manifests
+      run: make manifests
```

Comment:
there is technically another generate command too https://github.com/ngrok/ngrok-ingress-controller/blob/main/Makefile#L61 it was just created with kubebuilder though and I'm not sure we are even leveraging it until we make CRDs